lms/views/api/exceptions.py

```diff
@@ -103,10 +104,15 @@ def __init__(self, context, request):
         self.context = context
         self.request = request
 
+        # 400 is the default status for error responses.
+        # Some of the views below override this.
+        self.request.response.status_int = 400
+
     @exception_view_config(context=ValidationError)
     def validation_error(self):
-        return self.error_response(
-            422, message=self.context.explanation, details=self.context.messages
+        self.request.response.status_int = 422
```

Comment:
If a view doesn't want the default status code of `400` set by the `__init__()` method above then it just overwrites it.